charts/cosmos-watcher/values.yaml

```diff
@@ -1,70 +1,153 @@
-# Default values for cosmos-watcher.
-replicaCount: 1
+# Default values for cosmos-watcher
+
+# Name override (optional)
+nameOverride: ""
+
+# Full name override (optional)
+fullnameOverride: ""
 
-# Container configurations
-containers:
-  main:
-    image: ghcr.io/p2p-org/devops-cosmos-watcher
-    tag: v1.0.0
-    pullPolicy: Always
-    ports:
-      - name: http
-        containerPort: 8080
-        protocol: TCP
-    resources:
-      requests:
-        memory: 512Mi
-        cpu: 100m
-      limits:
-        memory: 512Mi
-        cpu: 100m
-    command: []
-    args: []
-    livenessProbe:
-      tcpSocket:
-        port: 8080
-      initialDelaySeconds: 60
-      periodSeconds: 20
-      timeoutSeconds: 5
-      failureThreshold: 3
-    readinessProbe:
-      tcpSocket:
-        port: 8080
-      periodSeconds: 5
-      timeoutSeconds: 1
-      failureThreshold: 3
-    volumeMounts: []
-    env:
-      - name: GITHUB_API_URL
-        value: https://raw.githubusercontent.com
-      - name: CHAIN_REGISTRY_BASE_URL
-        value: /cosmos/chain-registry/master
-      - name: SLACK_WEBHOOK_URL
-        valueFrom:
-          secretKeyRef:
-            name: slack-webhook-url
-            key: url
-      - name: POLLING_INTERVAL
-        value: 5m
-      - name: LOG_LEVEL
-        value: info
-      - name: CACHE_DURATION
-        value: 5m
-    envFrom: []
-
-imagePullSecrets:
-- name: github-secret
+# Replica count (StatefulSet typically uses 1 replica)
+replicaCount: 1
 
+# Image configuration
+image:
+  repository: ghcr.io/p2p-org/devops-cosmos-watcher
+  tag: latest
+  pullPolicy: IfNotPresent
+
+# Image pull secrets (for private registries like GHCR)
+# Example:
+# imagePullSecrets:
+#   - name: ghcr-secret
+imagePullSecrets: []
+
+# Resource limits
+resources:
+  requests:
+    memory: "128Mi"
+    cpu: "100m"
+  limits:
+    memory: "512Mi"
+    cpu: "500m"
+
+# Health probes
+livenessProbe:
+  httpGet:
+    path: /api/v1/health
+    port: 8080
+  initialDelaySeconds: 30
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 3
+
+readinessProbe:
+  httpGet:
+    path: /api/v1/health
+    port: 8080
+  initialDelaySeconds: 10
+  periodSeconds: 5
+  timeoutSeconds: 1
+  failureThreshold: 3
+
+# Environment variables
+env: []
+
+# Environment variables from ConfigMaps/Secrets (envFrom)
+# Example:
+# envFrom:
+#   - configMapRef:
+#       name: my-config
+#   - secretRef:
+#       name: my-secret
+envFrom: []
+
+# External Secrets configuration (for OCI secret store)
+externalSecrets:
+  - name: cosmos-watcher-secrets
+    refreshInterval: 1h
+    secretStoreRef:
+      kind: ClusterSecretStore
+      name: oci-secrets
+    target:
+      name: cosmos-watcher-secrets
+      creationPolicy: Owner
+    data:
+      - secretKey: SLACK_WEBHOOK_URL
+        remoteRef:
+          key: SLACK_WEBHOOK_URL
+      - secretKey: GITHUB_TOKEN
+        remoteRef:
+          key: GITHUB_API_SECRET
```

Comment:
Can we make sure we use Cosmos-sec service account for this purpose, would be helpful in case we want to include private repositories like Lombard and such.